cluster-autoscaler/core/scale_down.go

```diff
@@ -911,32 +922,34 @@ func (sd *ScaleDown) scheduleDeleteEmptyNodes(emptyNodes []*apiv1.Node, cp cloud
 					metrics.RegisterScaleDown(1, gpu.GetGpuTypeForMetrics(cp.GPULabel(), cp.GetAvailableGPUTypes(), nodeToDelete, nodeGroup), metrics.Unready)
 				}
 			}
-			confirmation <- deleteErr
+			confirmation <- emptyNodeDeletionConfirmation{node: node, err: deleteErr}
```

Comment:
Let's drop the 'empty' part from emptyNodeDeleteConfirmation (both the type and variables). It's sent for both empty and non-empty nodes.

Reply:
In my understanding, this method and this channel is only used for empty nodes, but no problem!

Reply:
Right, nevermind:/ Shorter name is still nice, but that's just a nit in this case.

Reply:
Done already :)